src/neuroglancer/sliceview/backend.ts

```diff
@@ -237,34 +221,18 @@ export abstract class VolumeChunkSource extends ChunkSource implements VolumeChu
   }
 }
 
-@registerSharedObject(SLICEVIEW_RENDERLAYER_RPC_ID)
-export class RenderLayer extends SharedObjectCounterpart implements RenderLayerInterface {
-  rpcId: number;
-  sources: VolumeChunkSource[][];
+export abstract class RenderLayer extends SharedObjectCounterpart implements RenderLayerInterface {
+  sources: SliceViewChunkSource[][];
   layerChanged = new NullarySignal();
-
```

Comment:
Per my previous comment about refactoring this, maybe this stuff can actually stay part of sliceview, rather than be duplicated.